NHLGames/Utilities/AdDetection.vb

```diff
@@ -171,10 +171,10 @@ Namespace Utilities
                 Where(Function(x) x.MainWindowTitle = "fd://0 - VLC media player" OrElse x.MainWindowTitle.ToLower().Contains(" @ ")).
                 Select(Function(x) x.Id)
             Dim mpc64Processes = Process.GetProcessesByName("MPC-HC64").
-                Where(Function(x) x.MainWindowTitle = "stdin" OrElse x.MainWindowTitle.ToLower().Contains(" @ ")).
+                Where(Function(x) x.MainWindowTitle = "stdin" OrElse x.MainWindowTitle.ToLower().Contains(" @ ") OrElse x.MainWindowTitle.ToLower().Contains("_complete.m3u8")).
```

Comment:
There is an option to show the full URL in the title in MPC so it includes "nhl.com" or "NHL_GAME_VIDEO" as you suggested for VLC.

However, since it's not the default, I went with "_complete.m3u8" since all the streams I checked didn't have the "-trimmed" part.

Changed it to ".m3u8" only, as you suggested.